upgrade/steps_helpers.go

```diff
@@ -511,18 +511,25 @@ var getExpectedTargetLatest = stepv2.Func21E("From Upstream Releases", func(ctx
 	name, upstreamOrg string) (*UpstreamUpgradeTarget, error) {
 	latest := stepv2.Cmd(ctx, "gh", "release", "list",
```

Comment:
We should use `--json` when possible. Tracking issue is https://github.com/cli/cli/issues/4572.